lumicks/pylake/force_calibration/touchdown.py

```diff
@@ -70,6 +72,15 @@ def piecewise_linear(x, x0, y0, k1, k2):
     return np.piecewise(x, [x < x0], [lambda x: k1 * (x - x0) + y0, lambda x: k2 * (x - x0) + y0])
 
 
+def f_test(sse_restricted, sse_unrestricted, num_data, num_pars_difference, num_pars_unrestricted):
+    """Determine p-value whether the bigger model describes the variance significantly better"""
+    nominator = (sse_restricted - sse_unrestricted) / num_pars_difference
+    denominator = sse_unrestricted / (num_data - num_pars_unrestricted)
+    f_statistic = nominator / denominator
```

Comment:
This fails sometimes, not sure if it's just my environment: 
`pylake/force_calibration/touchdown.py:79: RuntimeWarning: divide by zero encountered in double_scalars
    f_statistic = nominator / denominator`
    
```suggestion
    if denominator == 0:
        RuntimeWarning("Could not determine variance due to invalid model data".)
        denominator = 1
        
    f_statistic = nominator / denominator
```

Reply:
That's a fair point. The reason this happens is because the simulated data in the tests is noise free. It will never happen for real data. It would also happen if <= 4 data points were provided. I will add a warning for it, and filter that warning specifically in the tests that are affected.

Reply:
Sounds reasonable - certainly if it does not normally occur outside testing.